slack-message-sender.el

```diff
@@ -45,9 +45,50 @@
         (setq message-id 1)
       (cl-incf message-id))))
 
+(defun slack-escape-message (message)
+  "Escape '<,' '>' & '&' in MESSAGE."
+  (replace-regexp-in-string
+   ">" "&gt;"
+   (replace-regexp-in-string
+    "<" "&lt;"
+    (replace-regexp-in-string "&" "&amp;" message))))
+
+(defun slack-link-users (message team)
+  "Add links to all references to valid users in MESSAGE."
+  (replace-regexp-in-string
+   "@\\<\\([A-Za-z0-9.-_]+\\)"
+   #'(lambda (text)
+       (let* ((username (match-string 1 text))
+              (id (slack-user-get-id username team)))
+         (if id
+             (format "<@%s|%s>" id username)
+           text)))
+   message t))
+
+(defun slack-link-channels (message team)
```

Comment:
Well, that's embarassing. I evaluated each form, but didn't load the file as a whole … sloppy. 5de5010 fixes this.